src/backend/db/products.js

```diff
@@ -1,13 +1,10 @@
-import { v4 as uuid } from "uuid";
-
 /**
  * Product Database can be added here.
  * You can add products of your wish with different attributes
  * */
 
 export const products = [
 	{
-		_id: uuid(),
```

Comment:
Why did you remove id from here?

Reply:
Mockbee is internally generating an id, so for that I removed it from here, when you read the data i.e is GET request we will have an id in that.